src/cljs/athens/main/core.cljs

```diff
@@ -40,6 +40,7 @@
                                                    :nodeIntegration true
                                                    :worldSafeExecuteJavaScript true
                                                    :enableRemoteModule true
+                                                   :enableBlinkFeatures 'OverlayScrollbars'
```

Comment:
Could you add a comment here along the lines of "Remove when `scrollbar-gutter` is supported"?

Reply:
Done!